test/Microsoft.ServiceHub.Analyzers.Tests/Microsoft.ServiceHub.Analyzers.Tests.csproj

```diff
@@ -21,4 +21,8 @@
     <ProjectReference Include="..\..\src\Microsoft.ServiceHub.Framework\Microsoft.ServiceHub.Framework.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Using Include="Xunit" />
```

Comment:
Does this add an implicit using to all files in the project?

Reply:
Yes.

Reply:
Very cool